jobs/user-projects/general/nova-timetable.hcl

```diff
@@ -81,16 +85,39 @@ job "nova-timetable" {
       }
     }
 
-    task "redis-db" {
+    task "redis" {
       driver = "docker"
 
       config {
         image = "redis:latest"
-        ports = ["db"]
+        ports = ["redis"]
+      }
+    }
+
+    task "postgres" {
+      driver = "docker"
+
+      config {
+        image = "postgres:17.0-alpine"
+        ports = ["postgres"]
```

Comment:
I would calll this "db" rather than postgress to keep consistency with our other jobs